README.md

```diff
@@ -218,19 +218,17 @@ endpointUrl:
 ## Deploying
 There are two ways to configure the build system to deploy the CDR service to a cloud:
 - Edit the`deploy.bash` file. Set two variables near the top of the file:
-  - Set `SET_DOCKER_REG="ip:port"`
-  - Set `SET_DOCKER_W=/path/to/docker/wrapper/"`
+  - `SET_DOCKER_REG="ip:port"`
+  - `SET_DOCKER_W="/path/to/docker/wrapper/"`
 
 OR
 
 - Avoid editing a file in source control by exporting values:
-```bash
-export DOCKER_REGISTRY="ip:port"
-export DOCKER_WRAPPER="/path/to/docker/wrapper/"
-```
 
-After configuring the build system, run the gradle task `deploy` 
-```bash
-./gradlew deploy
-```
+      export DOCKER_REGISTRY="ip:port"
+      export DOCKER_WRAPPER="/path/to/docker/wrapper/"
```

Comment:
✏️ Why isn't this wrapped in "\`\`\`bash" and "\`\`\`"?

```markdown
- Avoid editing a file in source control by exporting values:

    ```bash
    export DOCKER_REGISTRY="ip:port"
    export DOCKER_WRAPPER="/path/to/docker/wrapper/"
    ```
```

Reply:
In markdown, any line indented with four spaces is formatted as code. Using the spaces gets the text to line-up correctly. Surrounding it with ``` left-justifies it.

Reply:
The snippet that I suggested works for me without left-justifying it. I would prefer to have the bash styling.